string ros_master_uri
---
bool res
